app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -46,6 +46,7 @@ CONFIG_LOG_MODE_DEFERRED=y
 CONFIG_LOG_FUNC_NAME_PREFIX_INF=y
 CONFIG_COMP_VOLUME_WINDOWS_FADE=y
 CONFIG_COMP_UP_DOWN_MIXER=y
+CONFIG_SAMPLE_KEYPHRASE=y
```

Comment:
@kv2019i are samples going in an overlay ?

Reply:
@lgirdwood I think we can keep them in board files for now. We currently don't have a separate samples overlay, and the extra overlays don't in general get CI coverage, so better to have this in one of the CI-coverated configurations.